### PR TITLE
(SIMP-2686) Add acceptance tests for base_*

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-2686 claims there are idempotency issues, this test resolves them.

The default suite was also broken and should work against updated
fixtures.

SIMP-2686 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
> RC release which will require a bump in version
An RC still isn't a real release, no version bump is required.

**Comment by nick-markowski** [comment]
The beta and RC are actual releases. Version bump required.

**Comment by trevor-vaughan** [comment]
@jeefberkey and @nick-markowski Yeah, we are actually publishing the RPMs. Try not to conflate the release of SIMP with a release of any particular module. When the modules are published as RPMs, they're tagged and released in Git. If a module isn't ready for release, it should be tagged with alpha/beta/rc itself.

We really need to start pushing out RPMs on a regular basis instead of waiting for a SIMP rollup.

File: spec/acceptance/suites/default/00_yum_spec.rb [hunk 1]
```diff
@@ -46,19 +46,18 @@
 
   let(:hieradata) {
     <<-EOM
----
+---simp_apache::rsync_server : '127.0.0.1'
```
Comment (lamawithonel) [inline spec/acceptance/suites/default/00_yum_spec.rb]: typo

**Pull request merged**